### PR TITLE
Foreground image + transparency + fixes

### LINKED ISSUE #85: Suggestion: Colors on either side of "splitter"
I think it would be cool to set a color for either side of a splitter - this could be used to create a distinct separation to fit a color scheme where no part of the screen is more obscured than any other when using transparency in the colors.

### LINKED ISSUE #86: Suggestion: foreground image instead of color (transparency gradient?)
I think it would be awesome to set a visualizer up so you have, for instance, a nighttime scene as the background image, but a daytime version of the same shot comes in in the shape of a wave. This would be especially cool if you can use a gradient of  transparency. I've done a mockup: https://media.discordapp.net/attachments/419703257355517967/1137820343323594813/image.png This depicts a background image with an altered version of itself used as a foreground image. It is also set to use a gradient between fully transparent at the base and fully opaque at the tips.

### LINKED ISSUE #100: Can't connect to audio
2023.8.1 just displays the new "Let's dance!" screen
terminal shows no errors:
_[AURA] Listening to commands.

(NickvisionCavalier.GNOME:2): Gtk-CRITICAL **: 00:46:29.876: gtk_widget_get_parent: assertion 'GTK_IS_WIDGET (widget)' failed

(NickvisionCavalier.GNOME:2): Gtk-CRITICAL **: 00:46:29.883: gtk_widget_add_css_class: assertion 'GTK_IS_WIDGET (widget)' failed_

Fresh install on KDE neon 5.27, Plasma 5.27.6 
Kernel Version: 5.19.0-50-generic (64-bit)
pulseaudio permission is granted

### LINKED ISSUE #103: Update to Cava 0.9.1
https://github.com/karlstav/cava/releases/tag/0.9.1

### LINKED ISSUE #111: Update Aura to 2023.9.1

### PR DESCRIPTION
**Pull request by fsobolev**
Main:
- [x] Ability to set foreground image (Closes #86)
- [x] Ability to set transparency for background and foreground images (Closes #85)

Other:
- [x] Update Aura (Closes #111)
- [x] Update CAVA in flatpak and snap (Closes #103)
- [x] Switch back to using Pulse Audio by default (Closes #100)

### CODE REVIEW

File: NickvisionCavalier.Shared/Models/Renderer.cs [hunk 1]
```diff
@@ -60,55 +70,100 @@ public void Draw(float[] sample, float width, float height)
...
+        if (_bgImageIndex != Configuration.Current.BgImageIndex)
+        {
+            _bgImageBitmap?.Dispose();
+            _bgTargetBitmap?.Dispose();
+            if (Configuration.Current.BgImageIndex != -1)
+            {
+                var images = new List<string>();
+                foreach (var file in Directory.GetFiles($"{UserDirectories.ApplicationConfig}{Path.DirectorySeparatorChar}images"))
+                {
+                    if (file.EndsWith(".jpg") || file.EndsWith(".jpeg") || file.EndsWith(".png"))
```
Comment (nlogozzo) [inline NickvisionCavalier.Shared/Models/Renderer.cs:91]: I'd do 
```csharp
using System.IO;
var extension = Path.GetExtension(file).ToLower();
if(extension == ".jpg"|| ...)
{
...
}
```

**Review by nlogozzo — CHANGES_REQUESTED** [review]
Also, `po` and `pot` needs updating

**Comment by nlogozzo** [comment]
Did you want to fix #99 and then make this rc1 and stable friday?

**Comment by fsobolev** [comment]
> Did you want to fix #99

Sorry, forgot to make a separate commit, it was fixed in https://github.com/NickvisionApps/Cavalier/pull/112/commits/024089f6daa2dea5fbbff5e1ac1f0d0e4782daf7

> make this rc1 and stable friday?

I want to make this beta1; rc and stable next week. Considering the amount of changes I want to give more time for people who want to test it.

**Review by nlogozzo — APPROVED** [review]

**Pull request merged**